hello_network/lib/hello_network.ex

```diff
@@ -1,18 +1,61 @@
 defmodule HelloNetwork do
+  @moduledoc """
+  Example of setting up Networking on a Nerves device.
+  """
 
-  alias Nerves.Networking
+  require Logger
 
-  @interface :eth0
+  alias Nerves.{Network}
```

Comment:
Let's not put the curly-braces in unless we have multiple things to alias. Newbies might be confused by this.